src/components/Text/index.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
```

Comment:
I don't like putting both of these components in the index file

Reply:
I'll put them into different files